.babelrc

```diff
@@ -1 +1,4 @@
-{ "presets": ["es2015"] }
+{
+  "presets": ["es2015"],
+  "plugins": ["add-module-exports"]
```

Comment:
if we use http://babeljs.io/docs/learn-es2015/#modules way to import module, could we omit this dependency?